examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/setuppayloadscanner/QrCodeInfo.kt

```diff
@@ -1,13 +1,13 @@
 package com.google.chip.chiptool.setuppayloadscanner
 
 import android.os.Parcelable
-import chip.setuppayload.OptionalQRCodeInfo.OptionalQRCodeInfoType
+import chip.onboardingpayload.OptionalQRCodeInfo.OptionalQRCodeInfoType
 import kotlinx.parcelize.Parcelize
 
 @Parcelize 
 data class QrCodeInfo(
     val tag: Int,
     val type: OptionalQRCodeInfoType,
-    val data: String,
+    val data: String?,
```

Comment:
Do we have any case in which QrCodeInfo contains null data? I though this part is mandatory

Reply:
https://github.com/project-chip/connectedhomeip/blob/a06bca80c1e214717c48d9fbe78a73fc2df42e31/src/controller/java/src/chip/onboardingpayload/OptionalQRCodeInfo.kt#L23

The part you implemented is nullable, so I made it the same.

Reply:
I think we should align java-matter-controller with Android CHIPTool QRCodeInfo object with a null data does not represent a valid state. I can update this in https://github.com/project-chip/connectedhomeip/pull/26963